.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
can you please delete this or add .vscode to the .gitignore, these should not get pushed